package.json

```diff
@@ -1267,11 +1267,9 @@
   ],
   "scripts": {
     "vscode:prepublish": "npm run esbuild-bundle",
-    "esbuild-base": "del-cli ./dist && esbuild \"./src/**/*.ts\" --outdir=dist --format=cjs --platform=node --target=node18",
-    "esbuild": "npm run esbuild-base",
-    "esbuild-watch": "npm run esbuild-base -- --watch",
-    "esbuild-bundle": "del-cli ./dist && esbuild ./src/extension.ts --bundle --outfile=dist/extension.js --external:vscode --format=cjs --platform=node --target=node18 --minify --sourcemap",
-    "compile": "tsc",
+    "esbuild": "del-cli ./dist && esbuild \"./src/**/*.ts\" --outdir=dist/src --format=cjs --platform=node --target=node18 --sourcemap",
```

Comment:
does it make sense to keep this around anymore? Use esbuild to bundle, tsc as pre launch task. We can have a single watch job whether running extension or tests

Reply:
Yeah that will definitely work, good catch! Let me fix it up.